alfresco-transform-misc/alfresco-transform-misc-boot/src/main/java/org/alfresco/transformer/Application.java

```diff
@@ -1,74 +1,74 @@
- * provided under the following open source license terms:
- * -
- * Alfresco is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- * -
- * Alfresco is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Lesser General Public License for more details.
- * -
- * You should have received a copy of the GNU Lesser General Public License
- * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
- * #L%
- */
-package org.alfresco.transformer;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.actuate.autoconfigure.metrics.MeterRegistryCustomizer;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
-import org.springframework.boot.context.event.ApplicationReadyEvent;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.event.EventListener;
-
-import io.micrometer.core.instrument.MeterRegistry;
-
-@SpringBootApplication
-@EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class})
-public class Application
-{
-    private static final Logger logger = LoggerFactory.getLogger(Application.class);
-
-    @Value("${container.name}")
-    private String containerName;
-
-    @Bean
-    MeterRegistryCustomizer<MeterRegistry> metricsCommonTags()
-    {
-        return registry -> registry.config().commonTags("containerName", containerName);
-    }
-
-    public static void main(String[] args)
-    {
-        SpringApplication.run(Application.class, args);
-    }
-
-    @EventListener(ApplicationReadyEvent.class)
-    public void startup()
-    {
-        logger.info("--------------------------------------------------------------------------------------------------------------------------------------------------------------");
-        logger.info("The transformers in this project use libraries from Apache. See the license at http://www.apache.org/licenses/LICENSE-2.0. or in /Apache\\\\ 2.0.txt");
-        logger.info("Additional libraries used:");
-        logger.info("* htmlparser http://htmlparser.sourceforge.net/license.html");
-        logger.info("--------------------------------------------------------------------------------------------------------------------------------------------------------------");
-
-        logger.info("Starting application components... Done");
-    }
-}
+/*
```

Comment:
Format only changes should be done in separate PRs - did anything else change other than line-endings (cc @eknizat @PunchedChimera ) ?

Reply:
No changes to this file.

Reply:
line changes fixed now set to CRLF